settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
 
 INSTALLED_APPS = [
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
     "db",
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` is set to `db.User`, which is correct for using the custom `User` model. Ensure that this matches the model path in your `db` app.